core/src/data-generator/custom-dataset.ts

```diff
@@ -0,0 +1,228 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
Can we provide as output the location of logs used by glue so a user can check potential issues/error with the job.

Reply:
stored as an object parameter and providing example to create a CFN output from it